docs/data-fundamentals/README.md

```diff
@@ -1,28 +1,16 @@
-# Data and DataOps fundamentals
```

Comment:
This file should be renamed `docs/data-fundamentals.md`

Reply:
I understand your thinking, and I was thinking the same - but if I move it outside the folder it will not show up as a section, instead it will move to the top of the list before the sprint-planning doc. which will be very confusing.  

In addition there may be more docs coming to this section, so I will leave this here for now and we can discuss more later.